.idea/.gitignore

```diff
@@ -0,0 +1,5 @@
+# Default ignored files
+/shelf/
```

Comment:
These are personal and should be included in your personal ~/.gitignore, not in the repo

Reply:
> It was hard to read this diff because your code formatter does not match the repository's. Once you fix the formatting, I can review more thoroughly since it won't show so many lines of diff.

I think in the future it would be better to configure eslint, prettier, editorconfig

Reply:
We have those but it's a bit complicated because it's in a monorepo, so those are in the parent directory. If you know a good solution for managing shared config in a monorepo, let me know.

Reply:
In the meantime:
```
{
  "trailingComma": "es5",
  "tabWidth": 2,
  "semi": true,
  "singleQuote": false,
  "endOfLine": "lf"
}
```